woodwork/tests/accessor/test_accessor_utils.py

```diff
@@ -56,8 +59,20 @@ def test_init_series_all_parameters(sample_series):
 
 
 def test_init_series_error_on_invalid_conversion(sample_series):
-    xfail_dask_and_koalas(sample_series)
+    xfail_koalas(sample_series)
+    if dd and isinstance(sample_series, dd.Series):
+        pytest.xfail('Dask type conversion with astype does not fail until compute is called')
     error_message = "Error converting datatype for sample_series from type object to type Int64. " \
         "Please confirm the underlying data is consistent with logical type Integer."
     with pytest.raises(TypeConversionError, match=error_message):
         init_series(sample_series, logical_type='integer')
+
+
+def test_is_series(sample_series, sample_df):
```

Comment:
I seem to remember needing to parameterize tests differently when they take in more than one fixture so they don't double up weirdly. 

For example--this produces 9 tests since it's getting every df-series pair. Should we do that here as well? Maybe just grab a single column from `sample_df` for the series check?

Reply:
Yes, you are correct. I think we also have other tests like this that need to be fixed. Probably should create an issue for that. For now, I like your idea to just use sample_df and pull a series off of that.

Reply:
I did a quick review for other tests that used multiple fixtures and didn't see the one I was thinking of...either I missed it or I was confused about other tests using this pattern. Regardless, I updated this test and if I come across others that use multiple fixtures that would create a matrix of tests unnecessarily, I'll create a new issue.

Reply:
Sounds good!